index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First Capstone</title> 
+    <link rel="stylesheet" href="./CSS/style.css">
+    <script src="script.js" defer></script>
+</head>
+<body>
+    <header>
+        <nav id="nav-wrap">
+            <div>
+                <ul class="get-intouch">
+                    <li><a href="https://web.facebook.com/gdgnairobi/?_rdc=1&_rdr"><img src="./img/facebook.png" alt="facebook-icon"></a></li>
+                    <li><a href="https://twitter.com/gdg_nairobi?lang=en"><img src="./img/twitter.png" alt="twitter-icon"></a></li>
+                    <li><a href="">English</a></li>
+                    <li><a href="">My Page</a></li>
+                    <li><a href="">Logout</a></li>
+                </ul>
+            </div>
+            <div class="hamburger">
+                <span class="bar bar1"></span>
+                <span class="bar bar2"></span>
+                <span class="bar bar3"></span>
+            </div>
+            <div class="main-menu">               
+                    <a class="nav-item nav-active" href="about.html">About</a>
+                     <a class="nav-item" href="">Program</a>
+                    <a  class="nav-item"  href="">Join</a>
+                    <a  class="nav-item" href="">Sponsor</a>
+                   <a  class="nav-item nav-items" href="">Sign in</a>
+            </div>
```

Comment:
- [ ] Hi. Great job, but you are required to have the logo of your app on the left-hand side of your nav. Kindly add it following the design provided
<img width="973" alt="image" src="https://user-images.githubusercontent.com/37901469/192102180-05c6b744-f94f-4a8e-9cf3-06712b4466bf.png">